example/src/main/scala/fommil/dda/main.scala

```diff
@@ -49,45 +56,94 @@ object Main extends SafeApp {
     } yield ()
   }.run.swallowError
 
-  // runs the app, requires that refresh tokens are provided
-  def agents(bearer: BearerToken): Task[Unit] = {
-    type HT[f[_], a] = EitherT[f, JsonClient.Error, a]
-    type GT[f[_], a] = StateT[f, BearerToken, a]
-    type FT[f[_], a] = StateT[f, WorldView, a]
-
-    type H[a] = HT[Task, a]
```

Comment:
this is the nice bit, it removes the manual handling of various `f[_]` embeddings